### PR TITLE
Add logging

### PR DESCRIPTION
**Pull request by ameech**
Add basic logging to give insight into what's going on.

### CODE REVIEW

File: src/Worker.php [hunk 1]
```diff
@@ -85,11 +96,14 @@ private function tick($queue)
             $result = call_user_func($handler, $message);
 
             $this->event->finish($message);
+            $this->logger->info(sprintf('`%s` job finished', $handle));
 
             if ($result === false) {
+                $this->logger->notice('shutting down');
```
Comment (elazar) [inline src/Worker.php]: Do we maybe want to include the name of the handler responsible for terminating the process here?

Comment (ameech) [inline src/Worker.php]: Ideas on a message for that?

Comment (elazar) [inline src/Worker.php]:
``` php
sprintf('shutting down by request of `%s`, $handle)
```

Comment (ameech) [inline src/Worker.php]: 👍

**Comment by ameech** [comment]
@elazar I've cleaned it up a little bit more by moving events/logging within the worker into their own methods.

**Comment by elazar** [comment]
👍

**Pull request merged**